napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -210,7 +213,7 @@ def _install(
             ]
 
         process = self._create_process(installer)
-        process.setArguments(cmd + list(pkg_list))
+        process.setArguments(cmd + list(pkg_list) + extra_deps)
```

Comment:
@jaimergp since napari conda packages might not be available, I added all the current napari dependencies when installing a plugin to preserve those constraints